test/integration/scheduler/queue_test.go

```diff
@@ -501,6 +500,63 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod1"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
+			name:         "Pod rejected by the interPodAffinity plugin is requeued when updating the node with the topologyKey label of the pod anti affinity",
+			initialNodes: []*v1.Node{st.MakeNode().Name("fake-node").Label("node", "fake-node").Obj()},
+			initialPods: []*v1.Pod{
```

Comment:
When we initially schedule pod1 on node1, pod2 will be affected by anit affinity and cannot be scheduled. When the node updates the topologyKey, it will trigger re-queuing. Is this understanding correct?

Reply:
Yes. That's what the test should be.